src/Okta.Sdk/CreateUserWithImportedHashedPasswordOptions.cs

```diff
@@ -0,0 +1,40 @@
+﻿// <copyright file="CreateUserWithImportedHashedPasswordOptions.cs" company="Okta, Inc">
+// Copyright (c) 2014 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace Okta.Sdk
+{
+    /// <summary>
+    /// Contains the options for creating a new <see cref="IUser">User</see> with imported hashed password.
+    /// Used with <see cref="IUsersClient.CreateUserAsync(CreateUserWithImportedHashedPasswordOptions, System.Threading.CancellationToken)"/>.
+    /// </summary>
+    /// <remarks>See <a href="https://developer.okta.com/docs/reference/api/users/#create-user-with-imported-hashed-password">Create User with Imported Hashed Password</a> in the documentation.</remarks>
+    public sealed class CreateUserWithImportedHashedPasswordOptions
+    {
+        /// <summary>
+        /// Gets or sets the new user's profile.
+        /// </summary>
+        /// <value>
+        /// The user's profile.
+        /// </value>
+        public UserProfile Profile { get; set; }
+
+        /// <summary>
+        /// Gets or sets the new user's hashed password
+        /// </summary>
+        /// <value>
+        /// The user's hashed password.
+        /// </value>
+        public PasswordCredentialHash Hash { get; set; }
```

Comment:
Should we call this property `PasswordCredentialHash` or `HashedPassword` instead of `Hash`?